src/IncludedCommands/Administrative/cmds.lua

```diff
@@ -44,11 +44,15 @@ return {
                         for _,Prefix in Prefixes do
                             local CommandString = Prefix..CmdrCommand.Name.." "
                             for _,Argument in CmdrCommand.Args do
-                                if Argument.Optional == true then
-                                    CommandString = CommandString.."("..Argument.Name..") "
-                                else
-                                    CommandString = CommandString..Argument.Name.." "
-                                end
+                                if typeof(Argument)=='function' then
```

Comment:
Formatting issues:
- Spaces are used surrounding `==` in every other case of the project.
- Double quotes are used for all other strings.

Requested line:
```lua
if typeof(Argument) == "function" then
```